README.md

```diff
@@ -208,390 +212,4 @@ var myBar = bar({name: 'Moe\'s'});
-
-The remaining arguments are passed to all `.init()`
-functions. **WARNING** Avoid using two different `.init()`
-functions that expect different arguments. `.init()`
-functions that take arguments should not be considered safe to
-compose with other `.init()` functions that also take
-arguments. Taking arguments with an `.init()` function is an
-anti-pattern that should be avoided, when possible.
-
-
-### stamp.static() ###
-
-Take n objects and add all its properties to the stamp (aka factory object).
-* @return {Object} stamp A new stamp.
-
-
-## Utility methods ##
-
-### stampit.methods() ###
-
-Shortcut for `stampit().methods()`
-
-### stampit.refs() ###
-
-Shortcut for `stampit().refs()` 
-
-### stampit.init() ###
-
-Shortcut for `stampit().init()`
-
-### stampit.props() ###
-
-Shortcut for `stampit().props()`
-
-### stampit.compose() ###
-
-Take two or more stamps produced from stampit() and
-combine them to produce a new stamp. Combining overrides
-properties with last-in priority.
-
-* `@param {...Function|Function[]} stamp` any number of stamps.
-* `@return {Function}` A new stamp composed from arguments.
-
-
-### stampit.mixin(), .extend(), .mixIn(), .assign() ###
-
-Aliases to `Object.assign()`. Deprecated.
-
-
-### stampit.isStamp(obj) ###
-
-Take an object and return `true` if it's a stamp, `false` otherwise.
-
-
-### stampit.convertConstructor() ###
-
-Take an old-fashioned JS constructor and return a stamp  that
-you can freely compose with other stamps. It is possible to
-use constructors that take arguments. Simply pass the arguments
-into the returned stamp after the properties object:
-`var myInstance = myStamp(props, arg1, arg2);`
-
-Note that if you use this feature, it is **not safe** to compose
-the resulting stamp with other stamps willy-nilly, because if two
-different stamps depend on the argument passing feature, the arguments
-will probably clash with each other, producing very unexpected results.
-
- * @param  {Function} Constructor
- * @return {Function} A composable stampit factory (aka stamp).
-
-```js
-  // The old constructor / class thing...
-  var Constructor = function Constructor() {
-    this.thing = 'initialized';
-  };
-  Constructor.prototype.foo = function foo() { return 'foo'; };
-
-  // The conversion
-  var oldskool = stampit.convertConstructor(Constructor);
-
-  // A new stamp to compose with...
-  var newskool = stampit({
-    methods: {
-      bar: function bar() { return 'bar'; }
-     // your methods here...
-    },
-    init: function () {
-      this.baz = 'baz';
-    }
-  });
-
-  // Now you can compose those old constructors just like you could
-  // with any other stamp...
-  var myThing = stampit.compose(oldskool, newskool);
-
-  var t = myThing();
-
-  t.thing; // 'initialized',
-
-  t.foo(); // 'foo',
-
-  t.bar(); // 'bar'
-```
+For more examples see [API](docs/API.md) and [advances examples](docs/advanced_examples.md).
```

Comment:
For more examples see the [API](docs/API.md) and the [advances examples](docs/advanced_examples.md).